grails-app/services/au/org/ala/volunteer/ForumService.groovy

```diff
@@ -405,6 +400,9 @@ class ForumService {
             it.delete(flush: true)
         }
 
+        // Remove message from topic
+        ForumTopic forumTopic = message.topic
+        forumTopic.removeFromMessages(message)
```

Comment:
Does forumTopic need to be saved here?

Reply:
Testing tells us no, but could add the save just to be sure?